cmd/juju/commands/debuglog.go

```diff
@@ -71,17 +71,25 @@ The filtering options combine as follows:
 
 The '--tail' option waits for and continuously prints new log lines after displaying the most recent log lines.
 
-The '--notail' option displays the most recent log lines and then exits immediately.
+The '--no-tail' option displays the most recent log lines and then exits immediately.
 
 The '--lines' and '--limit' options control the number of log lines displayed:
 * --lines option prints the specified number of the most recent lines and then waits for new lines. This implies --tail.
-* --limit option prints up to the specified number of the most recent lines and exits. This implies --notail.
+* --limit option prints up to the specified number of the most recent lines and exits. This implies --no-tail.
+* setting --lines or --limit to 0 will print the maximum number of recent lines available and allowed by the server.
```

Comment:
I don't think we need " and allowed by the server"

Reply:
Sure will update this thanks Ian!